image/templates/helm/stackrox-central/values.yaml.htpl

```diff
@@ -201,17 +201,22 @@
 #      maxConns: 90
 #      statementTimeoutMs: 1200000
 #
-#  # The admin password setting for communication with Central's DB.
-#  # When a value is set explicitly, this is always used, even on upgrade.
-#  # Otherwise, a password will be automatically generated if `generate` is set to true,
-#  # or left unset (null) and the Helm chart is being installed (as opposed to upgraded).
-#  # Should only be used when utilizing Postgres as central's DB
-#  password:
-#    value: null
-#    generate: null
-#
-#  postgresConfig: "@config/centraldb/postgresql.conf|config/centraldb/postgresql.conf.default"
-#  hbaConfig: "@config/centraldb/pg_hba.conf|config/centraldb/pg_hba.conf.default"
+#    # The admin password setting for communication with Central's DB.
+#    # When a value is set explicitly, this is always used, even on upgrade.
+#    # Otherwise, a password will be automatically generated if `generate` is set to true,
+#    # or left unset (null) and the Helm chart is being installed (as opposed to upgraded).
+#    # Should only be used when utilizing Postgres as central's DB
+#    password:
+#      value: null
+#      generate: null
+#
+#    postgresConfig: "@config/centraldb/postgresql.conf|config/centraldb/postgresql.conf.default"
+#    hbaConfig: "@config/centraldb/pg_hba.conf|config/centraldb/pg_hba.conf.default"
+#
+#    # Specifying configOverride mounts the specified config map in the same namespace which must contain
+#    # both pg_hba.conf and postgresql.conf. This should only be used when the default settings are not
+#    # sufficient and manual override is required.
+#    configOverride: null
```

Comment:
We need to carry this indention all the way to the scanner section.  For instance, based on this indentation `nodeSelector` at line 221 and at line 121 are at the same level.  Same for persistence.